src/app/technical-record/brakes/axle-brakes/axle-brakes.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit, ChangeDetectionStrategy, Input } from '@angular/core';
+import { Axle } from '@app/models/tech-record.model';
+import { VEHICLE_TYPES } from '@app/app.enums';
+
+@Component({
+  selector: 'vtm-axle-brakes',
+  templateUrl: './axle-brakes.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class AxleBrakesComponent implements OnInit {
+
+  @Input() vehicleType: string;
+  @Input() axle: Axle;
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  CanDisplay(): boolean {
+    return this.vehicleType === VEHICLE_TYPES.TRL;
```

Comment:
we should move this check to the brakes template

Reply:
moved.